### DISCUSSION TITLE
Which command can view the buffer list like ls in vim?

### DISCUSSION THREAD

**Discussion by kendling** [helix-editor/helix#10834] [Q&A] [answered]
Hi everyone, which command can view the buffer list like ls in vim?

**Comment by kirawi** [comment, marked as answer]
You can bring up the buffer picker via `space-b`

**Reply by kendling** [reply]
Thank you!

And then, I has another problem:

I saw this command  `:reset-diff-change, :diffget, :diffg`	Reset the diff change at the cursor position.

It it using in diff view?

How to enter diff view like vimdiff.

I can't found any command or commandline parameter for it.

**Reply by kirawi** [reply]
Not yet implemented https://github.com/helix-editor/helix/issues/405
